rpm/podman.spec

```diff
@@ -241,14 +247,18 @@ export CGO_CFLAGS+=" -m64 -mtune=generic -fcf-protection=full"
 
 export GOPROXY=direct
 
-LDFLAGS="-X %{ld_libpod}/define.buildInfo=${SOURCE_DATE_EPOCH:-$(date +%s)} \
+export LDFLAGS="-X %{ld_libpod}/define.buildInfo=${SOURCE_DATE_EPOCH:-$(date +%s)} \
          -X %{ld_libpod}/config._installPrefix=%{_prefix} \
          -X %{ld_libpod}/config._etcDir=%{_sysconfdir} \
          -X %{ld_project}/pkg/systemd/quadlet._binDir=%{_bindir}"
 
+export LDFLAGS_PODMAN='-X "%{ld_libpod}/define.buildOrigin=%{build_origin}"'
```

Comment:
why this extra definition? And the variable name certainly does not make much sense either. Functional speaking there is no difference between this and the above so why not simply add it directly to LDFLAGS?
If that is just to escape the quotes then `-X \"%{ld_libpod}/define.buildOrigin=%{build_origin}\"` should work there too

Reply:
done thanks.